src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_combined.py

```diff
@@ -147,7 +147,7 @@ def __call__(
         prompt: Optional[Union[str, List[str]]] = None,
         height: int = 512,
         width: int = 512,
-        prior_num_inference_steps: int = 60,
+        prior_num_inference_steps: int = 30,
```

Comment:
I think 60 is correct here, and it is the [docstring value](https://github.com/huggingface/diffusers/blob/7271f8b7170923944e193dac5b8513bbbb3b883f/src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_combined.py#L197) that needs to be changed? cc @kashif

Reply:
no lets switch it to 30, 60 seems too excessive right @dome272?

Reply:
ok got it!